flepimop/gempyor_pkg/tests/utils/test_read_df.py

```diff
@@ -38,13 +38,13 @@ def test_raises_not_implemented_error(self) -> None:
         """
         with pytest.raises(
             expected_exception=NotImplementedError,
-            match="Invalid extension txt. Must be 'csv' or 'parquet'.",
+            match=r".*Supported extensions are `.csv` or `.parquet`.",
```

Comment:
prefer match bad argument value rather than message (though also worth matching the supported argument values, to confirm we're informing users).

Reply:
(this comment repeats for a few more changes here and in test_utils)